samples/matter/common/src/app/matter_init.cpp

```diff
@@ -170,6 +178,20 @@ void DoInitChipServer(intptr_t /* unused */)
 	SetDeviceInstanceInfoProvider(sLocalInitData.mFactoryDataProvider);
 	SetDeviceAttestationCredentialsProvider(sLocalInitData.mFactoryDataProvider);
 	SetCommissionableDataProvider(sLocalInitData.mFactoryDataProvider);
+
+#ifdef CONFIG_NCS_SAMPLE_MATTER_TEST_EVENT_TRIGGERS
+	/* Read the enable key from the factory data set */
+	uint8_t enableKeyData[chip::TestEventTriggerDelegate::kEnableKeyLength] = {};
+	MutableByteSpan enableKey(enableKeyData);
+	sInitResult = sLocalInitData.mFactoryDataProvider->GetEnableKey(enableKey);
+	VerifyInitResultOrReturn(sInitResult, "GetEnableKey() failed");
+	Nrf::Matter::TestEventTrigger::Instance().SetEnableKey(enableKey);
+	VerifyInitResultOrReturn(sInitResult, "SetEnableKey() failed");
```

Comment:
Do you mean, the part of the Matter's core?

Reply:
Oh, you meant our default triggers implementation :) Hmm, the only reason is to avoid hiding it from the users, because they need to read the enable key and register it in the TestEventTrigger class. So if we move it to default event triggers it won't be shown explicitly in the init method.

Reply:
discussed f2f